setup.py

```diff
@@ -230,3 +243,19 @@ def run(self):
     # Installing as zip files would break due to references to __file__
     zip_safe=False
 )
+
+
+def main():
+    """Invoke installation process using setuptools."""
+    warnings.filterwarnings(
+        'ignore',
+        message="Unknown distribution option: '\w+'",
```

Comment:
I think we want to match the specific options that we're utilizing here (project_urls and python_requires).  Using this more general regex would mean that we are not warned if we typo an option name in a future change (like ```autor_email='info@ansible.com',``` )